heat/examples/vm_two_nics_with_share.yaml

```diff
@@ -74,17 +74,7 @@ resources:
             key_name: { get_param: user_key }
             image: { get_param: user_vm_image }
             flavor: { get_param: user_vm_flavor }
-            user_data: |
-                #!/usr/bin/env python
-                import os,json,re,requests
-                r = requests.get('http://169.254.169.254/openstack/latest/meta_data.json')
-                data = r.json()
-                m = re.search('\'(.+?)\'', data['meta']['share_export_loc_path'])
-                sharepath = m.group(1)
-                mountpoint = '/mnt/share'
-                os.mkdir(mountpoint)
-                os.chown(mountpoint,0,0)
-                os.system("mount %s %s" % (sharepath,mountpoint))
+            user_data: { get_file: 'https://raw.githubusercontent.com/StephaneGerardVUB/openstack-templates/HeatTemplateWithShare/heat/examples/mount_share.py' }
```

Comment:
don't reference your own fork, use the main repo (it will only work once this PR is merged)

Reply:
also do not use `*.py`    extension, cloud-init is a bit stupid and only executes `*.sh`  scripts

Reply:
@StephaneGerardVUB I have tested this PR directly from our openstack dashboard and it works! the only thing that we should change as @stdweird said is to change this URL by the final URL, do not use your personal github repo.
Also move the script within scripts directory. This will help us to keep yaml files and scripts in different places, it is cleaner and we can reuse the scripts quite easy.

Reply:
and I was mistaken `*.py`  also works :-) cloud-init is quite agnostic about that